public/media-hunt/generaldata.js

```diff
@@ -0,0 +1,151 @@
+const normalization = (data) => ({
```

Comment:
This file should only return this "normalization" function, and parse it correctly.
Also remove all the boilerplate/test data from this file.

Reply:
done